app/pages/admin/project-status.cjsx

```diff
@@ -110,7 +150,30 @@ ProjectExperimentalFeatures = React.createClass
               <input type="checkbox" name={task} checked={@setting(task)} onChange={@updateTasks.bind @, task} />
               {task.charAt(0).toUpperCase() + task.slice(1)}
             </label>}
-          </div>
+        </div>
+      </AutoSave>
+      <br />
+      <h4>Project Tags</h4>
+      <AutoSave resource={@props.project}>
+        <span className="form-label">Discipline Tag</span>
+        <br />
+        <Select
+          ref="disciplineSelect"
+          name="disciplines"
+          placeholder="Add Discipline Tag"
+          className="discipline-tag"
+          value={@state.disciplineTagList}
+          options={DISCIPLINES}
+          multi={true}
+          onChange={@handleDisciplineTagChange} />
+        <small className="form-help">Enter or select one or more discipline tags to identify which field(s) of research your project belongs to. These tags will determine the categories your project will appear under on the main Zooniverse projects page, if your project becomes a full Zooniverse project. </small>
+      </AutoSave>
+      <br />
+      <AutoSave resource={@props.project}>
+        <span className="form-label">Other Tags</span>
+        <br />
+        <TagSearch name="tags" multi={true} value={@state.otherTagList} onChange={@handleOtherTagChange} />
+        <small className="form-help">Enter a list of additional tags to describe your project separated by commas to help users find your project.</small>
```

Comment:
maybe some wording on how you can use spaces in your tags - i found the interface slightly non-intuitive to use spaces / commas for delimiting.